pandera/hypotheses.py

```diff
@@ -4,12 +4,19 @@
 from typing import Callable, Union, Optional, List, Dict
 
 import pandas as pd
-from scipy import stats
+
+try:
+    from scipy import stats
+except ImportError as exc:  # pragma: no cover
+    raise RuntimeError((
```

Comment:
It's better to reraise an `ImportError`. It's more precise than a `RuntimeError`. Same in `pandera.io`.